src/main/_docs/use-che-as-an-ide/ide-commands.md

```diff
@@ -58,10 +58,10 @@ fi
 | `${explorer.current.file.relpath}` | Path relative to the `/projects` folder in project tree   
 | `${explorer.current.project.name}` | Project name of the file currently selected in explorer   
 | `${explorer.current.project.type}` | Project type of the file currently selected in explorer   
-| `${server.<name>}` | Returns protocol, hostname and port of an internal server. `<name>` is defined by the same of the internal service that you have exposed in your workspace recipe. <br><br> Returns the hostname and port of a service or application you launch inside of a machine. <br><br> The hostname resolves to the hostname or the IP address of the workspace machine. This name varies depending upon where Docker is running and whether it is embedded within a VM.  See [Networking]({{base}}{{site.links["setup-configuration"]}}). <br><br> The port returns the Docker ephemeral port that you can give to your external clients to connect to your internal service. Docker uses ephemeral port mapping to expose a range of ports that your clients may use to connect to your internal service. This port mapping is dynamic.   
-| `${server.<name>.protocol}` | Returns protocol of a server registered by name   
-| `${server.<name>.hostname}` | Returns hostname of a server registered by name   
-| `${server.<name>.port}` | Returns port of a server registered by name
+| `${server.<port>}` | Returns protocol, hostname and port of an internal server. `<port>` is defined by the same internal port of the internal service that you have exposed in your workspace recipe. <br><br> Returns the hostname and port of a service or application you launch inside of a machine. <br><br> The hostname resolves to the hostname or the IP address of the workspace machine. This name varies depending upon where Docker is running and whether it is embedded within a VM.  See [Networking]({{base}}{{site.links["setup-configuration"]}}). <br><br> The port returns the Docker ephemeral port that you can give to your external clients to connect to your internal service. Docker uses ephemeral port mapping to expose a range of ports that your clients may use to connect to your internal service. This port mapping is dynamic.   
+| `${server.<port>.protocol}` | Returns protocol of a server registered by name   
+| `${server.<port>.hostname}` | Returns hostname of a server registered by name   
+| `${server.<port>.port}` | Returns port of a server registered by name
```

Comment:
This seems weird because it's port.port.  Is there some other way for us to do that?